### ISSUE TITLE
Aws Lambda usage: Cannot convert undefined or null to object

### ISSUE THREAD

**Issue by PinnacleOne** [szokodiakos/typegoose#207]
**typegoose version: 5.4.1
mongoose version: 5.3.2
reflect-metadata version: 0.1.12
nodejs vesrsion: 8.11
typescript version: 2.9.2
webpack version: 4.5.0
serverless-webpack version: 5.1.1**

I'm unable to run this library on AWS Lambda service.

When I try just simply create model:
```
new MyCollection().getModelForClass(MyCollection, {
      existingMongoose: mongoose,
      schemaOptions: {
        collection: 'my_collection'
      },
      existingConnection: mongoose.connection
    });
```
it throw error:

> TypeError: Cannot convert undefined or null to object
> at Function.keys (<anonymous>)
> at m.keys [as add] (/var/task/webpack:/node_modules/mongoose/lib/schema.js:351:1)
> at c.add [as buildSchema] (/var/task/webpack:/node_modules/typegoose/lib/typegoose.js:57:1)
> at c.buildSchema [as setModelForClass] (/var/task/webpack:/node_modules/typegoose/lib/typegoose.js:34:1)
> at c.setModelForClass (/var/task/webpack:/node_modules/typegoose/lib/typegoose.js:21:1)
> at getModelForClass (/var/task/webpack:/handler.ts:34:36)
> at Generator.next (<anonymous>)
> at value (/var/task/webpack:/node_modules/buffer-from/index.js:66:1)
> at new Promise (<anonymous>)
> at value (/var/task/webpack:/node_modules/buffer-from/index.js:66:1)
> at __awaiter (/var/task/webpack:/handler.ts:33:53)
> at body (/var/task/webpack:/handler.ts:21:26)
> at Generator.next (<anonymous>)
> at value (/var/task/webpack:/node_modules/buffer-from/index.js:66:1)
> at <anonymous>
> at process._tickDomainCallback (internal/process/next_tick.js:228:7)

How to resolve such issue? Because is useless for serverless framework now :(

On serverless-offline I don't have this issue everything works fine. It only happens when I run code in deployed function of AWS Lambda.

**Comment by kktos** [comment]
That's the same as #91 
Got it too :/

**Comment by andreialecu** [comment]
Getting the same error and it seems to be because of webpack production mode/uglifyjs/minification.

If I disable webpack minification it works.

**Comment by andreialecu** [comment]
See https://github.com/serverless-heaven/serverless-webpack/issues/345

**Comment by hasezoey** [comment]
this can be closed because its like #242 

@Ben305
